edit-post/assets/stylesheets/main.scss

```diff
@@ -194,12 +194,25 @@ body.gutenberg-editor-page {
 		&:checked:focus {
 			box-shadow: 0 0 0 2px $dark-gray-500;
 		}
+	}
+
+	input[type="checkbox"] {
+		border-radius: $radius-round-rectangle / 2;
 
 		&::before {
 			margin: -4px 0 0 -5px;
 			color: $white;
 		}
 	}
+
+	input[type="radio"] {
+		border-radius: 50%;
```

Comment:
Can you use `$radius-round` here, from https://github.com/WordPress/gutenberg/blob/master/edit-post/assets/stylesheets/_variables.scss?